### PR TITLE
fix graph execute when packaged

### LINKED ISSUE #1040: [Theme]Fail to see 'Graph/JSON' in Default Dark mode
**OS:** Windows, Mac, Linux
**Build Version:** [20190213.2](https://dev.azure.com/ms-azuretools/AzCode/_build/results?buildId=3146)
**Regression:** Previous Release

**Repro Steps:**
1. Make sure the theme is Default Dark mode.
2. Open a Graph Explorer.
3. Check the view.

**Expect:**
Easy to see all contents.

**Actual:**
Fail to see 'Graph/JSON'.
image

**More Info:**
This issue also reproduces when entering a query.
image

### LINKED ISSUE #1041: [Suggestion]The textbox border is visible
**OS:** Windows, Mac, Linux
**Build Version:** [20190213.2](https://dev.azure.com/ms-azuretools/AzCode/_build/results?buildId=3146)
**Regression**: Not a Regression

**Repro Steps:**
1. Open a Graph Explorer.
2. Check the textbox.

**Actual:**
The textbox border is invisible till clicking it.
image

**Suggestion:**
The textbox border is visible.

### LINKED ISSUE #1043: 'Execute' button doesn't work in Graph Explorer
**OS:** Windows, Mac, Linux
**Build Version:** [20190213.2](https://dev.azure.com/ms-azuretools/AzCode/_build/results?buildId=3146)
**Regression:** Previous Release

**Repro Steps:**
1. Open a Graph Explorer.
2. Check whether the 'Execute' button works well or not.

**Expect:**
'Execute' button works well.

**Actual:**
'Execute' button doesn't work.
image

### PR DESCRIPTION
**Pull request by StephenWeatherford**
Fixes #1043, fixes #1041, fixes #1040

1) Fix references to graphClient.js and d3. Only repros when packaged.
2) Fix color issues in vscode Insiders (they've removed --color and --background-color -> use --vscode-editor-{background,forecolor} instead)

### CODE REVIEW

**Comment by StephenWeatherford** [comment]
@PrashanthCorp It would be great if you could get to this soon so I can send off to CTI again tonight.  Thanks!

File: webpack.config.js [hunk 1]
```diff
@@ -105,17 +108,22 @@ let config = dev.getDefaultWebpackConfig({
         }
     ], // end of loaderRules
 
+
     plugins: [
         // Replace vscode-languageserver/lib/files.js with a modified version that doesn't have webpack issues
         new webpack.NormalModuleReplacementPlugin(
+
```
Comment (PrashanthCorp) [inline webpack.config.js]: Stray newline?

File: webpack.config.js [hunk 2]
```diff
@@ -105,17 +108,22 @@ let config = dev.getDefaultWebpackConfig({
...
             /[/\\]vscode-languageserver[/\\]lib[/\\]files\.js/,
             require.resolve('./build/vscode-languageserver-files-stub.js')
         ),
 
         // Copy files to dist folder where the runtime can find them
         new CopyWebpackPlugin([
             // getCoreNodeModule.js -> dist/node_modules/getCoreNodeModule.js
-            { from: './src/utils/getCoreNodeModule.js', to: 'node_modules' }
+            { from: './src/utils/getCoreNodeModule.js', to: 'node_modules' },
+
```
Comment (PrashanthCorp) [inline webpack.config.js]: Stray newline?

Comment (StephenWeatherford) [reply webpack.config.js]: No, I'm keeping them separated.

**Pull request merged**